contracts/libraries/Errors.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+error ErrInvalidThreshold(bytes4 msgSig);
+error ErrOnlySelfCall(bytes4 msgSig);
+error ErrUnauthorized(bytes4 msgSig);
+error ErrEmptyArrayLength();
```

Comment:
Should be `ErrEmptyArray` or `ErrZeroLengthArray`.

Reply:
solved.